Build failure: mozillavpn

### Nixpkgs version

- Unstable (25.11)

### Steps to reproduce

`nix-build -A mozillavpn`

### Can Hydra reproduce this build failure?

Hydra doesn’t try to build the package.

### Link to Hydra build job

_No response_

### Relevant log output

```console
this derivation will be built:
  /nix/store/yrd68n0z2p1lwmcvb0h0xh61vbnj20w6-mozillavpn-2.31.1.drv
building '/nix/store/yrd68n0z2p1lwmcvb0h0xh61vbnj20w6-mozillavpn-2.31.1.drv'...
Running phase: qtPreHook
Running phase: unpackPhase
unpacking source archive /nix/store/87czd54js860kmp1bydg74bp3skl09ar-source
source root is source
Executing cargoSetupPostUnpackHook
Finished cargoSetupPostUnpackHook
Running phase: patchPhase
patching script interpreter paths in scripts/utils/xlifftool.py
scripts/utils/xlifftool.py: interpreter directive changed from "#! /usr/bin/env python3" to "/nix/store/cfapjd2rvqrpry4grb0kljnp8bvnvfxz-python3-3.13.8/bin/python3"
Executing cargoSetupPostPatchHook
Validating consistency between /build/source/Cargo.lock and /build/cargo-deps-vendor/Cargo.lock
Finished cargoSetupPostPatchHook
Running phase: updateAutotoolsGnuConfigScriptsPhase
Updating Autotools / GNU config script to a newer upstream version: ./3rdparty/sentry/external/breakpad/autotools/config.sub
Updating Autotools / GNU config script to a newer upstream version: ./3rdparty/sentry/external/breakpad/autotools/config.guess
Running phase: configurePhase
cmake flags: -DCMAKE_FIND_USE_SYSTEM_PACKAGE_REGISTRY=OFF -DCMAKE_FIND_USE_PACKAGE_REGISTRY=OFF -DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON -DCMAKE_BUILD_TYPE=Release -DBUILD_TESTING=OFF -DCMAKE_INSTALL_LOCALEDIR=/nix/store/r7k2r2lmnrdnswziyv342avf44831lbb-mozillavpn-2.31.1/share/locale -DCMAKE_INSTALL_LIBEXECDIR=/nix/store/r7k2r2lmnrdnswziyv342avf44831lbb-mozillavpn-2.31.1/libexec -DCMAKE_INSTALL_LIBDIR=/nix/store/r7k2r2lmnrdnswziyv342avf44831lbb-mozillavpn-2.31.1/lib -DCMAKE_INSTALL_DOCDIR=/nix/store/r7k2r2lmnrdnswziyv342avf44831lbb-mozillavpn-2.31.1/share/doc/Mozilla -DCMAKE_INSTALL_INFODIR=/nix/store/r7k2r2lmnrdnswziyv342avf44831lbb-mozillavpn-2.31.1/share/info -DCMAKE_INSTALL_MANDIR=/nix/store/r7k2r2lmnrdnswziyv342avf44831lbb-mozillavpn-2.31.1/share/man -DCMAKE_INSTALL_INCLUDEDIR=/nix/store/r7k2r2lmnrdnswziyv342avf44831lbb-mozillavpn-2.31.1/include -DCMAKE_INSTALL_SBINDIR=/nix/store/r7k2r2lmnrdnswziyv342avf44831lbb-mozillavpn-2.31.1/sbin -DCMAKE_INSTALL_BINDIR=/nix/store/r7k2r2lmnrdnswziyv342avf44831lbb-mozillavpn-2.31.1/bin -DCMAKE_INSTALL_NAME_DIR=/nix/store/r7k2r2lmnrdnswziyv342avf44831lbb-mozillavpn-2.31.1/lib -DCMAKE_POLICY_DEFAULT_CMP0025=NEW -DCMAKE_FIND_FRAMEWORK=LAST -DCMAKE_STRIP=/nix/store/x8mydcgbry214s802nzvy7fdljx404ym-gcc-wrapper-14.3.0/bin/strip -DCMAKE_RANLIB=/nix/store/x8mydcgbry214s802nzvy7fdljx404ym-gcc-wrapper-14.3.0/bin/ranlib -DCMAKE_AR=/nix/store/x8mydcgbry214s802nzvy7fdljx404ym-gcc-wrapper-14.3.0/bin/ar -DCMAKE_C_COMPILER=gcc -DCMAKE_CXX_COMPILER=g++ -DCMAKE_INSTALL_PREFIX=/nix/store/r7k2r2lmnrdnswziyv342avf44831lbb-mozillavpn-2.31.1 -DQT_LCONVERT_EXECUTABLE=/nix/store/fwmh85x9612hr8hvsx3dg313m380bsai-qttools-6.10.0-dev/bin/lconvert -DQT_LUPDATE_EXECUTABLE=/nix/store/fwmh85x9612hr8hvsx3dg313m380bsai-qttools-6.10.0-dev/bin/lupdate -DQT_LRELEASE_EXECUTABLE=/nix/store/fwmh85x9612hr8hvsx3dg313m380bsai-qttools-6.10.0-dev/bin/lrelease -DQT_OPTIONAL_TOOLS_PATH=/nix/store/qld4r900hwnf2qq2pl6v1811zgpj4i51-qttools-6.10.0 -DQT_OPTIONAL_TOOLS_PATH=/nix/store/qld4r900hwnf2qq2pl6v1811zgpj4i51-qttools-6.10.0 -DQT_OPTIONAL_TOOLS_PATH=/nix/store/qld4r900hwnf2qq2pl6v1811zgpj4i51-qttools-6.10.0
Configuring for Unix Makefiles
APP_VERSION=2.31.1
-- The C compiler identification is GNU 14.3.0
-- The CXX compiler identification is GNU 14.3.0
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /nix/store/x8mydcgbry214s802nzvy7fdljx404ym-gcc-wrapper-14.3.0/bin/gcc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /nix/store/x8mydcgbry214s802nzvy7fdljx404ym-gcc-wrapper-14.3.0/bin/g++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- clang-tidy: Binary not found. Linting will be skipped.
Generated BUILD_ID: 2.198001010000
-- Performing Test HAVE__Wall
-- Performing Test HAVE__Wall - Success
-- Performing Test HAVE__Wextra
-- Performing Test HAVE__Wextra - Success
-- Performing Test HAVE__Waggregate_return
-- Performing Test HAVE__Waggregate_return - Success
-- Performing Test HAVE__Wcast_align
-- Performing Test HAVE__Wcast_align - Success
-- Performing Test HAVE__Wcast_qual
-- Performing Test HAVE__Wcast_qual - Success
-- Performing Test HAVE__Wconversion
-- Performing Test HAVE__Wconversion - Success
-- Performing Test HAVE__Wdeclaration_after_statement
-- Performing Test HAVE__Wdeclaration_after_statement - Success
-- Performing Test HAVE__Wdouble_promotion
-- Performing Test HAVE__Wdouble_promotion - Success
-- Performing Test HAVE__Wfloat_equal
-- Performing Test HAVE__Wfloat_equal - Success
-- Performing Test HAVE__Wformat_security
-- Performing Test HAVE__Wformat_security - Success
-- Performing Test HAVE__Winit_self
-- Performing Test HAVE__Winit_self - Success
-- Performing Test HAVE__Wjump_misses_init
-- Performing Test HAVE__Wjump_misses_init - Success
-- Performing Test HAVE__Wlogical_op
-- Performing Test HAVE__Wlogical_op - Success
-- Performing Test HAVE__Wmissing_braces
-- Performing Test HAVE__Wmissing_braces - Success
-- Performing Test HAVE__Wmissing_declarations
-- Performing Test HAVE__Wmissing_declarations - Success
-- Performing Test HAVE__Wmissing_format_attribute
-- Performing Test HAVE__Wmissing_format_attribute - Success
-- Performing Test HAVE__Wmissing_include_dirs
-- Performing Test HAVE__Wmissing_include_dirs - Success
-- Performing Test HAVE__Wmissing_prototypes
-- Performing Test HAVE__Wmissing_prototypes - Success
-- Performing Test HAVE__Wnested_externs
-- Performing Test HAVE__Wnested_externs - Success
-- Performing Test HAVE__Wno_coverage_mismatch
-- Performing Test HAVE__Wno_coverage_mismatch - Success
-- Performing Test HAVE__Wold_style_definition
-- Performing Test HAVE__Wold_style_definition - Success
-- Performing Test HAVE__Wpacked
-- Performing Test HAVE__Wpacked - Success
-- Performing Test HAVE__Wpedantic
-- Performing Test HAVE__Wpedantic - Success
-- Performing Test HAVE__Wpointer_arith
-- Performing Test HAVE__Wpointer_arith - Success
-- Performing Test HAVE__Wredundant_decls
-- Performing Test HAVE__Wredundant_decls - Success
-- Performing Test HAVE__Wshadow
-- Performing Test HAVE__Wshadow - Success
-- Performing Test HAVE__Wsign_conversion
-- Performing Test HAVE__Wsign_conversion - Success
-- Performing Test HAVE__Wstrict_overflow
-- Performing Test HAVE__Wstrict_overflow - Success
-- Performing Test HAVE__Wstrict_prototypes
-- Performing Test HAVE__Wstrict_prototypes - Success
-- Performing Test HAVE__Wtrampolines
-- Performing Test HAVE__Wtrampolines - Success
-- Performing Test HAVE__Wundef
-- Performing Test HAVE__Wundef - Success
-- Performing Test HAVE__Wunreachable_code
-- Performing Test HAVE__Wunreachable_code - Success
-- Performing Test HAVE__Wunused
-- Performing Test HAVE__Wunused - Success
-- Performing Test HAVE__Wvariadic_macros
-- Performing Test HAVE__Wvariadic_macros - Success
-- Performing Test HAVE__Wvla
-- Performing Test HAVE__Wvla - Success
-- Performing Test HAVE__Wwrite_strings
-- Performing Test HAVE__Wwrite_strings - Success
-- Performing Test HAVE__Wimplicit_fallthrough_3
-- Performing Test HAVE__Wimplicit_fallthrough_3 - Success
-- Performing Test HAVE__Werror_implicit_int
-- Performing Test HAVE__Werror_implicit_int - Success
-- Performing Test HAVE__Werror_implicit_function_declaration
-- Performing Test HAVE__Werror_implicit_function_declaration - Success
-- Performing Test HAVE__Werror_partial_availability
-- Performing Test HAVE__Werror_partial_availability - Failed
-- Performing Test HAVE__Qunused_arguments
-- Performing Test HAVE__Qunused_arguments - Failed
-- Performing Test HAVE__Wno_long_long
-- Performing Test HAVE__Wno_long_long - Success
-- Performing Test HAVE__fno_omit_frame_pointer
-- Performing Test HAVE__fno_omit_frame_pointer - Success
-- Performing Test HAVE__O0
-- Performing Test HAVE__O0 - Success
-- Performing Test HAVE__Wall_CXX
-- Performing Test HAVE__Wall_CXX - Success
-- Performing Test HAVE__Wextra_CXX
-- Performing Test HAVE__Wextra_CXX - Success
-- Performing Test HAVE__Wcast_align_CXX
-- Performing Test HAVE__Wcast_align_CXX - Success
-- Performing Test HAVE__Wformat_security_CXX
-- Performing Test HAVE__Wformat_security_CXX - Success
-- Performing Test HAVE__Wmissing_declarations_CXX
-- Performing Test HAVE__Wmissing_declarations_CXX - Success
-- Performing Test HAVE__Wmissing_format_attribute_CXX
-- Performing Test HAVE__Wmissing_format_attribute_CXX - Success
-- Performing Test HAVE__Wpacked_bitfield_compat_CXX
-- Performing Test HAVE__Wpacked_bitfield_compat_CXX - Success
-- Performing Test HAVE__Wredundant_decls_CXX
-- Performing Test HAVE__Wredundant_decls_CXX - Success
-- Performing Test HAVE__Wvla_CXX
-- Performing Test HAVE__Wvla_CXX - Success
-- Performing Test HAVE__Wno_unused_parameter_CXX
-- Performing Test HAVE__Wno_unused_parameter_CXX - Success
-- Performing Test HAVE__Qunused_arguments_CXX
-- Performing Test HAVE__Qunused_arguments_CXX - Failed
-- Performing Test HAVE__Wconversion_CXX
-- Performing Test HAVE__Wconversion_CXX - Success
-- Performing Test HAVE__Wfloat_equal_CXX
-- Performing Test HAVE__Wfloat_equal_CXX - Success
-- Performing Test HAVE__Wsign_conversion_CXX
-- Performing Test HAVE__Wsign_conversion_CXX - Success
-- Performing Test HAVE__fno_omit_frame_pointer_CXX
-- Performing Test HAVE__fno_omit_frame_pointer_CXX - Success
-- Performing Test HAVE__O0_CXX
-- Performing Test HAVE__O0_CXX - Success
-- Looking for res_servicename
-- Looking for res_servicename - not found
-- Looking for res_servicename in resolv
-- Looking for res_servicename in resolv - not found
-- Looking for gethostbyname
-- Looking for gethostbyname - found
-- Looking for socket
-- Looking for socket - found
-- Looking for clock_gettime
-- Looking for clock_gettime - found
-- Looking for include file AvailabilityMacros.h
-- Looking for include file AvailabilityMacros.h - not found
-- Looking for include file sys/types.h
-- Looking for include file sys/types.h - found
-- Looking for include file sys/random.h
-- Looking for include file sys/random.h - found
-- Looking for include file sys/socket.h
-- Looking for include file sys/socket.h - found
-- Looking for include file sys/sockio.h
-- Looking for include file sys/sockio.h - not found
-- Looking for include file arpa/inet.h
-- Looking for include file arpa/inet.h - found
-- Looking for include file arpa/nameser_compat.h
-- Looking for include file arpa/nameser_compat.h - found
-- Looking for include file arpa/nameser.h
-- Looking for include file arpa/nameser.h - found
-- Looking for include file assert.h
-- Looking for include file assert.h - found
-- Looking for include file errno.h
-- Looking for include file errno.h - found
-- Looking for include file fcntl.h
-- Looking for include file fcntl.h - found
-- Looking for include file inttypes.h
-- Looking for include file inttypes.h - found
-- Looking for include file limits.h
-- Looking for include file limits.h - found
-- Looking for include file malloc.h
-- Looking for include file malloc.h - found
-- Looking for include file memory.h
-- Looking for include file memory.h - found
-- Looking for include file netdb.h
-- Looking for include file netdb.h - found
-- Looking for include file netinet/in.h
-- Looking for include file netinet/in.h - found
-- Looking for include file netinet6/in6.h
-- Looking for include file netinet6/in6.h - not found
-- Looking for include files sys/socket.h, net/if.h
-- Looking for include files sys/socket.h, net/if.h - found
-- Looking for include file signal.h
-- Looking for include file signal.h - found
-- Looking for include file socket.h
-- Looking for include file socket.h - not found
-- Looking for include file stdbool.h
-- Looking for include file stdbool.h - found
-- Looking for include file stdint.h
-- Looking for include file stdint.h - found
-- Looking for include file stdlib.h
-- Looking for include file stdlib.h - found
-- Looking for include file strings.h
-- Looking for include file strings.h - found
-- Looking for include file string.h
-- Looking for include file string.h - found
-- Looking for include file stropts.h
-- Looking for include file stropts.h - not found
-- Looking for include file sys/ioctl.h
-- Looking for include file sys/ioctl.h - found
-- Looking for include file sys/param.h
-- Looking for include file sys/param.h - found
-- Looking for include file sys/select.h
-- Looking for include file sys/select.h - found
-- Looking for include file sys/stat.h
-- Looking for include file sys/stat.h - found
-- Looking for include file sys/time.h
-- Looking for include file sys/time.h - found
-- Looking for include file sys/uio.h
-- Looking for include file sys/uio.h - found
-- Looking for include file sys/event.h
-- Looking for include file sys/event.h - not found
-- Looking for include file sys/epoll.h
-- Looking for include file sys/epoll.h - found
-- Looking for include file ifaddrs.h
-- Looking for include file ifaddrs.h - found
-- Looking for include file time.h
-- Looking for include file time.h - found
-- Looking for include file poll.h
-- Looking for include file poll.h - found
-- Looking for include file dlfcn.h
-- Looking for include file dlfcn.h - found
-- Looking for include file unistd.h
-- Looking for include file unistd.h - found
-- Looking for include files sys/types.h, netinet/tcp.h
-- Looking for include files sys/types.h, netinet/tcp.h - found
-- Performing Test HAVE_SOCKLEN_T
-- Performing Test HAVE_SOCKLEN_T - Success
-- Performing Test HAVE_TYPE_SOCKET
-- Performing Test HAVE_TYPE_SOCKET - Failed
-- Performing Test HAVE_SSIZE_T
-- Performing Test HAVE_SSIZE_T - Success
-- Performing Test HAVE_LONGLONG
-- Performing Test HAVE_LONGLONG - Success
-- Performing Test HAVE_STRUCT_ADDRINFO
-- Performing Test HAVE_STRUCT_ADDRINFO - Success
-- Performing Test HAVE_STRUCT_IN6_ADDR
-- Performing Test HAVE_STRUCT_IN6_ADDR - Success
-- Performing Test HAVE_STRUCT_SOCKADDR_IN6
-- Performing Test HAVE_STRUCT_SOCKADDR_IN6 - Success
-- Performing Test HAVE_STRUCT_SOCKADDR_STORAGE
-- Performing Test HAVE_STRUCT_SOCKADDR_STORAGE - Success
-- Performing Test HAVE_STRUCT_TIMEVAL
-- Performing Test HAVE_STRUCT_TIMEVAL - Success
-- Looking for AF_INET6
-- Looking for AF_INET6 - found
-- Looking for O_NONBLOCK
-- Looking for O_NONBLOCK - found
-- Looking for FIONBIO
-- Looking for FIONBIO - found
-- Looking for SIOCGIFADDR
-- Looking for SIOCGIFADDR - found
-- Looking for MSG_NOSIGNAL
-- Looking for MSG_NOSIGNAL - found
-- Looking for PF_INET6
-- Looking for PF_INET6 - found
-- Looking for SO_NONBLOCK
-- Looking for SO_NONBLOCK - not found
-- Looking for CLOCK_MONOTONIC
-- Looking for CLOCK_MONOTONIC - found
-- Performing Test HAVE_STRUCT_SOCKADDR_IN6_SIN6_SCOPE_ID
-- Performing Test HAVE_STRUCT_SOCKADDR_IN6_SIN6_SCOPE_ID - Success
-- Looking for strnlen
-- Looking for strnlen - found
-- Looking for memmem
-- Looking for memmem - found
-- Looking for closesocket
-- Looking for closesocket - not found
-- Looking for CloseSocket
-- Looking for CloseSocket - not found
-- Looking for connect
-- Looking for connect - found
-- Looking for connectx
-- Looking for connectx - not found
-- Looking for fcntl
-- Looking for fcntl - found
-- Looking for freeaddrinfo
-- Looking for freeaddrinfo - found
-- Looking for getaddrinfo
-- Looking for getaddrinfo - found
-- Looking for getenv
-- Looking for getenv - found
-- Looking for gethostname
-- Looking for gethostname - found
-- Looking for getnameinfo
-- Looking for getnameinfo - found
-- Looking for getrandom
-- Looking for getrandom - found
-- Looking for getservbyport_r
-- Looking for getservbyport_r - found
-- Looking for getservbyname_r
-- Looking for getservbyname_r - found
-- Looking for gettimeofday
-- Looking for gettimeofday - found
-- Looking for if_indextoname
-- Looking for if_indextoname - found
-- Looking for if_nametoindex
-- Looking for if_nametoindex - found
-- Looking for ConvertInterfaceIndexToLuid
-- Looking for ConvertInterfaceIndexToLuid - not found
-- Looking for ConvertInterfaceLuidToNameA
-- Looking for ConvertInterfaceLuidToNameA - not found
-- Looking for NotifyIpInterfaceChange
-- Looking for NotifyIpInterfaceChange - not found
-- Looking for RegisterWaitForSingleObject
-- Looking for RegisterWaitForSingleObject - not found
-- Looking for inet_net_pton
-- Looking for inet_net_pton - not found
-- Looking for inet_ntop
-- Looking for inet_ntop - found
-- Looking for inet_pton
-- Looking for inet_pton - found
-- Looking for ioctl
-- Looking for ioctl - found
-- Looking for ioctlsocket
-- Looking for ioctlsocket - not found
-- Looking for IoctlSocket
-- Looking for IoctlSocket - not found
-- Looking for recv
-- Looking for recv - found
-- Looking for recvfrom
-- Looking for recvfrom - found
-- Looking for send
-- Looking for send - found
-- Looking for sendto
-- Looking for sendto - found
-- Looking for setsockopt
-- Looking for setsockopt - found
-- Looking for socket
-- Looking for socket - found
-- Looking for strcasecmp
-- Looking for strcasecmp - found
-- Looking for strcmpi
-- Looking for strcmpi - not found
-- Looking for strdup
-- Looking for strdup - found
-- Looking for stricmp
-- Looking for stricmp - not found
-- Looking for strncasecmp
-- Looking for strncasecmp - found
-- Looking for strncmpi
-- Looking for strncmpi - not found
-- Looking for strnicmp
-- Looking for strnicmp - not found
-- Looking for writev
-- Looking for writev - found
-- Looking for arc4random_buf
-- Looking for arc4random_buf - found
-- Looking for stat
-- Looking for stat - found
-- Looking for getifaddrs
-- Looking for getifaddrs - found
-- Looking for poll
-- Looking for poll - found
-- Looking for pipe
-- Looking for pipe - found
-- Looking for pipe2
-- Looking for pipe2 - found
-- Looking for kqueue
-- Looking for kqueue - not found
-- Looking for epoll_create1
-- Looking for epoll_create1 - found
-- Looking for __system_property_get
-- Looking for __system_property_get - not found
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Success
-- Found Threads: TRUE
-- Looking for include file pthread.h
-- Looking for include file pthread.h - found
-- Looking for include file pthread_np.h
-- Looking for include file pthread_np.h - not found
-- Looking for pthread_init
-- Looking for pthread_init - not found
-- Performing Test HAVE_STDATOMIC
-- Performing Test HAVE_STDATOMIC - Success
-- Found WrapAtomic: TRUE
-- Found OpenGL: /nix/store/as2rg79vk4z7frb7hy27nmadc3fp1a4j-libglvnd-1.7.0/lib/libOpenGL.so
-- Found WrapOpenGL: TRUE
-- Found WrapVulkanHeaders: /nix/store/icyb0ffm7i4sajzphnf818firkqlx71h-vulkan-headers-1.4.328.0/include
Using Qt version 6.10.0
-- Found OpenSSL: /nix/store/llswcygvgv9x2sa3z6j7i0g5iqqmn5gn-openssl-3.6.0/lib/libcrypto.so (found version "3.6.0")
Building sentry for Linux
-- Found PkgConfig: /nix/store/0skggkamysbh0ksc3582lb69wzrpabcg-pkg-config-wrapper-0.29.2/bin/pkg-config (found version "0.29.2")
-- Checking for module 'libcap'
--   Found libcap, version 2.76
-- Checking for module 'libsecret-1'
--   Found libsecret-1, version 0.21.7
Package sysprof-capture-4 was not found in the pkg-config search path.
Perhaps you should add the directory containing `sysprof-capture-4.pc'
to the PKG_CONFIG_PATH environment variable
Package 'sysprof-capture-4', required by 'glib-2.0', not found
Package sysprof-capture-4 was not found in the pkg-config search path.
Perhaps you should add the directory containing `sysprof-capture-4.pc'
to the PKG_CONFIG_PATH environment variable
Package 'sysprof-capture-4', required by 'glib-2.0', not found
Package sysprof-capture-4 was not found in the pkg-config search path.
Perhaps you should add the directory containing `sysprof-capture-4.pc'
to the PKG_CONFIG_PATH environment variable
Package 'sysprof-capture-4', required by 'glib-2.0', not found
Package sysprof-capture-4 was not found in the pkg-config search path.
Perhaps you should add the directory containing `sysprof-capture-4.pc'
to the PKG_CONFIG_PATH environment variable
Package 'sysprof-capture-4', required by 'glib-2.0', not found
failed to initialize build cache at /homeless-shelter/.cache/go-build: mkdir /homeless-shelter: permission denied
failed to initialize build cache at /homeless-shelter/.cache/go-build: mkdir /homeless-shelter: permission denied
failed to initialize build cache at /homeless-shelter/.cache/go-build: mkdir /homeless-shelter: permission denied
failed to initialize build cache at /homeless-shelter/.cache/go-build: mkdir /homeless-shelter: permission denied
-- Checking for module 'systemd'
--   Found systemd, version 258
CMake Warning (dev) at /nix/store/r7wdw4nkkjii690wwnzkgj68cg4shh2v-qtbase-6.10.0/lib/cmake/Qt6Core/Qt6CoreMacros.cmake:3472 (message):
  Qt policy QTP0004 is not set: You need qmldir files for each extra
  directory that contains .qml files for your module.  Check
  https://doc.qt.io/qt-6/qt-cmake-policy-qtp0004.html for policy details.
  Use the qt_policy command to set the policy and suppress this warning.

Call Stack (most recent call first):
  /nix/store/rz0lc8irp5knfzwakm86p3ifkflr1w1i-qtdeclarative-6.10.0/lib/cmake/Qt6Qml/Qt6QmlMacros.cmake:3897 (__qt_internal_setup_policy)
  /nix/store/rz0lc8irp5knfzwakm86p3ifkflr1w1i-qtdeclarative-6.10.0/lib/cmake/Qt6Qml/Qt6QmlMacros.cmake:960 (qt6_target_qml_sources)
  /nix/store/rz0lc8irp5knfzwakm86p3ifkflr1w1i-qtdeclarative-6.10.0/lib/cmake/Qt6Qml/Qt6QmlMacros.cmake:1405 (qt6_add_qml_module)
  src/ui/CMakeLists.txt:5 (qt_add_qml_module)
This warning is for project developers.  Use -Wno-dev to suppress it.

CMake Warning at /nix/store/r7wdw4nkkjii690wwnzkgj68cg4shh2v-qtbase-6.10.0/lib/cmake/Qt6/QtPublicWalkLibsHelpers.cmake:285 (message):
  The QmlPrivate target is mentioned as a dependency for shared-sources, but
  not declared.  The linking might be incomplete.
Call Stack (most recent call first):
  /nix/store/r7wdw4nkkjii690wwnzkgj68cg4shh2v-qtbase-6.10.0/lib/cmake/Qt6/QtPublicWalkLibsHelpers.cmake:337 (__qt_internal_walk_libs)
  /nix/store/r7wdw4nkkjii690wwnzkgj68cg4shh2v-qtbase-6.10.0/lib/cmake/Qt6Core/Qt6CoreMacros.cmake:772 (__qt_internal_collect_all_target_dependencies)
  /nix/store/r7wdw4nkkjii690wwnzkgj68cg4shh2v-qtbase-6.10.0/lib/cmake/Qt6Core/Qt6CoreMacros.cmake:877 (_qt_internal_finalize_executable)
  /nix/store/r7wdw4nkkjii690wwnzkgj68cg4shh2v-qtbase-6.10.0/lib/cmake/Qt6Core/Qt6CoreMacros.cmake:1003 (qt6_finalize_target)
  src/CMakeLists.txt:81 (qt_finalize_target)


CMake Warning at /nix/store/rz0lc8irp5knfzwakm86p3ifkflr1w1i-qtdeclarative-6.10.0/lib/cmake/Qt6Qml/Qt6QmlMacros.cmake:4659 (message):
  The qml plugin 'qtgraphicaleffectsplugin' is a dependency of 'mozillavpn',
  but the link target it defines (Qt6::qtgraphicaleffectsplugin) does not
  exist in the current scope.  The plugin will not be linked.
Call Stack (most recent call first):
  /nix/store/r7wdw4nkkjii690wwnzkgj68cg4shh2v-qtbase-6.10.0/lib/cmake/Qt6Core/Qt6CoreMacros.cmake:799 (qt6_import_qml_plugins)
  /nix/store/r7wdw4nkkjii690wwnzkgj68cg4shh2v-qtbase-6.10.0/lib/cmake/Qt6Core/Qt6CoreMacros.cmake:799 (cmake_language)
  /nix/store/r7wdw4nkkjii690wwnzkgj68cg4shh2v-qtbase-6.10.0/lib/cmake/Qt6Core/Qt6CoreMacros.cmake:877 (_qt_internal_finalize_executable)
  /nix/store/r7wdw4nkkjii690wwnzkgj68cg4shh2v-qtbase-6.10.0/lib/cmake/Qt6Core/Qt6CoreMacros.cmake:1003 (qt6_finalize_target)
  src/CMakeLists.txt:81 (qt_finalize_target)


CMake Warning at /nix/store/rz0lc8irp5knfzwakm86p3ifkflr1w1i-qtdeclarative-6.10.0/lib/cmake/Qt6Qml/Qt6QmlMacros.cmake:4659 (message):
  The qml plugin 'qtgraphicaleffectsprivateplugin' is a dependency of
  'mozillavpn', but the link target it defines
  (Qt6::qtgraphicaleffectsprivate) does not exist in the current scope.  The
  plugin will not be linked.
Call Stack (most recent call first):
  /nix/store/r7wdw4nkkjii690wwnzkgj68cg4shh2v-qtbase-6.10.0/lib/cmake/Qt6Core/Qt6CoreMacros.cmake:799 (qt6_import_qml_plugins)
  /nix/store/r7wdw4nkkjii690wwnzkgj68cg4shh2v-qtbase-6.10.0/lib/cmake/Qt6Core/Qt6CoreMacros.cmake:799 (cmake_language)
  /nix/store/r7wdw4nkkjii690wwnzkgj68cg4shh2v-qtbase-6.10.0/lib/cmake/Qt6Core/Qt6CoreMacros.cmake:877 (_qt_internal_finalize_executable)
  /nix/store/r7wdw4nkkjii690wwnzkgj68cg4shh2v-qtbase-6.10.0/lib/cmake/Qt6Core/Qt6CoreMacros.cmake:1003 (qt6_finalize_target)
  src/CMakeLists.txt:81 (qt_finalize_target)


CMake Warning (dev) at /nix/store/r7wdw4nkkjii690wwnzkgj68cg4shh2v-qtbase-6.10.0/lib/cmake/Qt6Core/Qt6CoreMacros.cmake:3472 (message):
  Qt policy QTP0004 is not set: You need qmldir files for each extra
  directory that contains .qml files for your module.  Check
  https://doc.qt.io/qt-6/qt-cmake-policy-qtp0004.html for policy details.
  Use the qt_policy command to set the policy and suppress this warning.

Call Stack (most recent call first):
  /nix/store/rz0lc8irp5knfzwakm86p3ifkflr1w1i-qtdeclarative-6.10.0/lib/cmake/Qt6Qml/Qt6QmlMacros.cmake:3897 (__qt_internal_setup_policy)
  /nix/store/rz0lc8irp5knfzwakm86p3ifkflr1w1i-qtdeclarative-6.10.0/lib/cmake/Qt6Qml/Qt6QmlMacros.cmake:960 (qt6_target_qml_sources)
  /nix/store/rz0lc8irp5knfzwakm86p3ifkflr1w1i-qtdeclarative-6.10.0/lib/cmake/Qt6Qml/Qt6QmlMacros.cmake:1405 (qt6_add_qml_module)
  nebula/ui/components/CMakeLists.txt:5 (qt_add_qml_module)
This warning is for project developers.  Use -Wno-dev to suppress it.

CMake Warning (dev) at /nix/store/r7wdw4nkkjii690wwnzkgj68cg4shh2v-qtbase-6.10.0/lib/cmake/Qt6Core/Qt6CoreMacros.cmake:3472 (message):
  Qt policy QTP0004 is not set: You need qmldir files for each extra
  directory that contains .qml files for your module.  Check
  https://doc.qt.io/qt-6/qt-cmake-policy-qtp0004.html for policy details.
  Use the qt_policy command to set the policy and suppress this warning.

Call Stack (most recent call first):
  /nix/store/rz0lc8irp5knfzwakm86p3ifkflr1w1i-qtdeclarative-6.10.0/lib/cmake/Qt6Qml/Qt6QmlMacros.cmake:3897 (__qt_internal_setup_policy)
  /nix/store/rz0lc8irp5knfzwakm86p3ifkflr1w1i-qtdeclarative-6.10.0/lib/cmake/Qt6Qml/Qt6QmlMacros.cmake:960 (qt6_target_qml_sources)
  /nix/store/rz0lc8irp5knfzwakm86p3ifkflr1w1i-qtdeclarative-6.10.0/lib/cmake/Qt6Qml/Qt6QmlMacros.cmake:1405 (qt6_add_qml_module)
  nebula/ui/compat/CMakeLists.txt:5 (qt_add_qml_module)
This warning is for project developers.  Use -Wno-dev to suppress it.

Creating library translations from /build/source/src/translations
Importing translations: bg - completeness: 1.0
Importing translations: co - completeness: 0.9980694980694981
Importing translations: cs - completeness: 1.0
Importing translations: cy - completeness: 1.0
Importing translations: da - completeness: 1.0
Importing translations: de - completeness: 1.0
Importing translations: dsb - completeness: 1.0
Importing translations: el - completeness: 1.0
Importing translations: en - completeness: 1.0
Importing translations: en_CA - completeness: 0.9980694980694981
Importing translations: en_GB - completeness: 1.0
Importing translations: es_AR - completeness: 1.0
Importing translations: es_CL - completeness: 1.0
Importing translations: es_ES - completeness: 0.9980694980694981
Importing translations: es_MX - completeness: 0.9575289575289575
Importing translations: fa - completeness: 0.0019305019305019305
Importing translations: fi - completeness: 0.9961389961389961
Importing translations: fr - completeness: 1.0
Importing translations: fur - completeness: 0.9980694980694981
Importing translations: fy_NL - completeness: 1.0
Importing translations: hsb - completeness: 1.0
Importing translations: hu - completeness: 1.0
Importing translations: ia - completeness: 1.0
Importing translations: id - completeness: 1.0
Importing translations: is - completeness: 0.9903474903474904
Importing translations: it - completeness: 1.0
Importing translations: ja - completeness: 0.9266409266409267
Importing translations: ko - completeness: 1.0
Importing translations: lo - completeness: 1.0
Importing translations: nl - completeness: 1.0
Importing translations: oc - completeness: 0.14864864864864866
Importing translations: pa_IN - completeness: 0.9131274131274131
Importing translations: pl - completeness: 1.0
Importing translations: pt_BR - completeness: 0.9980694980694981
Importing translations: pt_PT - completeness: 0.9903474903474904
Importing translations: ru - completeness: 1.0
Importing translations: scn - completeness: 0.9980694980694981
Importing translations: sk - completeness: 1.0
Importing translations: sl - completeness: 1.0
Importing translations: sq - completeness: 0.9903474903474904
Importing translations: sv_SE - completeness: 0.0
Importing translations: th - completeness: 0.9768339768339769
Importing translations: tr - completeness: 1.0
Importing translations: uk - completeness: 0.9980694980694981
Importing translations: vi - completeness: 1.0
Importing translations: zh_CN - completeness: 1.0
Importing translations: zh_TW - completeness: 1.0
CMake Warning at /nix/store/rz0lc8irp5knfzwakm86p3ifkflr1w1i-qtdeclarative-6.10.0/lib/cmake/Qt6Qml/Qt6QmlMacros.cmake:4659 (message):
  The qml plugin 'qtgraphicaleffectsplugin' is a dependency of 'qml_tests',
  but the link target it defines (Qt6::qtgraphicaleffectsplugin) does not
  exist in the current scope.  The plugin will not be linked.
Call Stack (most recent call first):
  /nix/store/r7wdw4nkkjii690wwnzkgj68cg4shh2v-qtbase-6.10.0/lib/cmake/Qt6Core/Qt6CoreMacros.cmake:799 (qt6_import_qml_plugins)
  /nix/store/r7wdw4nkkjii690wwnzkgj68cg4shh2v-qtbase-6.10.0/lib/cmake/Qt6Core/Qt6CoreMacros.cmake:799 (cmake_language)
  /nix/store/r7wdw4nkkjii690wwnzkgj68cg4shh2v-qtbase-6.10.0/lib/cmake/Qt6Core/Qt6CoreMacros.cmake:877 (_qt_internal_finalize_executable)
  /nix/store/r7wdw4nkkjii690wwnzkgj68cg4shh2v-qtbase-6.10.0/lib/cmake/Qt6Core/Qt6CoreMacros.cmake:921:EVAL:1 (qt6_finalize_target)
  tests/qml/CMakeLists.txt:DEFERRED


CMake Warning at /nix/store/rz0lc8irp5knfzwakm86p3ifkflr1w1i-qtdeclarative-6.10.0/lib/cmake/Qt6Qml/Qt6QmlMacros.cmake:4659 (message):
  The qml plugin 'qtgraphicaleffectsprivateplugin' is a dependency of
  'qml_tests', but the link target it defines
  (Qt6::qtgraphicaleffectsprivate) does not exist in the current scope.  The
  plugin will not be linked.
Call Stack (most recent call first):
  /nix/store/r7wdw4nkkjii690wwnzkgj68cg4shh2v-qtbase-6.10.0/lib/cmake/Qt6Core/Qt6CoreMacros.cmake:799 (qt6_import_qml_plugins)
  /nix/store/r7wdw4nkkjii690wwnzkgj68cg4shh2v-qtbase-6.10.0/lib/cmake/Qt6Core/Qt6CoreMacros.cmake:799 (cmake_language)
  /nix/store/r7wdw4nkkjii690wwnzkgj68cg4shh2v-qtbase-6.10.0/lib/cmake/Qt6Core/Qt6CoreMacros.cmake:877 (_qt_internal_finalize_executable)
  /nix/store/r7wdw4nkkjii690wwnzkgj68cg4shh2v-qtbase-6.10.0/lib/cmake/Qt6Core/Qt6CoreMacros.cmake:921:EVAL:1 (qt6_finalize_target)
  tests/qml/CMakeLists.txt:DEFERRED


Creating library translations-unit-tests from /build/source/tests/unit_tests/translations
-- Configuring done (38.2s)
CMake Error at src/cmake/linux.cmake:41 (target_link_libraries):
  Target "mozillavpn" links to:

    Qt6::GuiPrivate

  but the target was not found.  Possible reasons include:

    * There is a typo in the target name.
    * A find_package call is missing for an IMPORTED target.
    * An ALIAS target is missing.

Call Stack (most recent call first):
  src/CMakeLists.txt:76 (include)


CMake Error at src/cmake/sentry.cmake:33 (target_link_libraries):
  The link interface of target "shared-sources" contains:

    Qt6::QmlPrivate

  but the target was not found.  Possible reasons include:

    * There is a typo in the target name.
    * A find_package call is missing for an IMPORTED target.
    * An ALIAS target is missing.

Call Stack (most recent call first):
  src/cmake/shared-sources.cmake:175 (include)
  src/CMakeLists.txt:45 (include)


-- Generating done (0.8s)
CMake Warning:
  Manually-specified variables were not used by the project:

    CMAKE_EXPORT_NO_PACKAGE_REGISTRY
    CMAKE_POLICY_DEFAULT_CMP0025


CMake Generate step failed.  Build files cannot be regenerated correctly.
error: Cannot build '/nix/store/yrd68n0z2p1lwmcvb0h0xh61vbnj20w6-mozillavpn-2.31.1.drv'.
       Reason: builder failed with exit code 1.
       Output paths:
         /nix/store/r7k2r2lmnrdnswziyv342avf44831lbb-mozillavpn-2.31.1
       Last 25 log lines:
       > CMake Error at src/cmake/sentry.cmake:33 (target_link_libraries):
       >   The link interface of target "shared-sources" contains:
       >
       >     Qt6::QmlPrivate
       >
       >   but the target was not found.  Possible reasons include:
       >
       >     * There is a typo in the target name.
       >     * A find_package call is missing for an IMPORTED target.
       >     * An ALIAS target is missing.
       >
       > Call Stack (most recent call first):
       >   src/cmake/shared-sources.cmake:175 (include)
       >   src/CMakeLists.txt:45 (include)
       >
       > 
       > -- Generating done (0.8s)
       > CMake Warning:
       >   Manually-specified variables were not used by the project:
       >
       >     CMAKE_EXPORT_NO_PACKAGE_REGISTRY
       >     CMAKE_POLICY_DEFAULT_CMP0025
       >
       > 
       > CMake Generate step failed.  Build files cannot be regenerated correctly.
       For full logs, run:
         nix log /nix/store/yrd68n0z2p1lwmcvb0h0xh61vbnj20w6-mozillavpn-2.31.1.drv
```

### Additional context

_No response_

### System metadata

 - system: `"x86_64-linux"`
 - host os: `Linux 6.17.2, NixOS, 25.11 (Xantusia), 25.11.20251015.544961d`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.31.2`
 - nixpkgs: `/nix/store/rcsmdj802sk3w9l8m0b0bvdy0jiw2wgr-source`


### Notify maintainers

@andersk 

---

**Note for maintainers:** Please tag this issue in your pull request description. (i.e. `Resolves #ISSUE`.)


### I assert that this issue is relevant for Nixpkgs

- [x] I assert that this is a bug and not a support request.
- [x] I assert that this is not a [duplicate of an existing issue](https://github.com/NixOS/nixpkgs/issues?q=is%3Aopen+is%3Aissue+label%3A%220.kind%3A+build+failure%22). 
- [x] I assert that I have read the [NixOS Code of Conduct](https://github.com/NixOS/.github/blob/master/CODE_OF_CONDUCT.md) and agree to abide by it.

### Is this issue important to you?

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/pkgs/by-name/mo/mozillavpn/package.nix
+++ b/pkgs/by-name/mo/mozillavpn/package.nix
@@ -4,6 +4,7 @@
   cargo,
   cmake,
   fetchFromGitHub,
+  fetchpatch,
   go,
   lib,
   libcap,
@@ -22,15 +23,21 @@
 
 stdenv.mkDerivation (finalAttrs: {
   pname = "mozillavpn";
-  version = "2.31.1";
+  version = "2.32.0";
   src = fetchFromGitHub {
     owner = "mozilla-mobile";
     repo = "mozilla-vpn-client";
     tag = "v${finalAttrs.version}";
     fetchSubmodules = true;
-    hash = "sha256-+Aexwj/iNIhriiUrMhtj2c9S3e5jGN7OvREXUzi/03s=";
+    hash = "sha256-STp/BCh3gELF0UgkMF2uUV9U5JgTNsqoh+Cog8fQy2c=";
   };
-  patches = [ ];
+  patches = [
+    # VPN-7309: Qt 6.10 QML loading fixes (#10832)
+    (fetchpatch {
+      url = "https://github.com/mozilla-mobile/mozilla-vpn-client/commit/5e7a26efd5acc3cdeeda8d1954459bff1a7e373e.patch";
+      hash = "sha256-CdvEuASPNYzQwyCMKXWZObOHH55WRFsxGYlEP8b20Mc=";
+    })
+  ];
 
   netfilter = buildGoModule {
     pname = "${finalAttrs.pname}-netfilter";

